src/dakara_feeder/__main__.py

```diff
@@ -0,0 +1,4 @@
+from dakara_feeder.commands import feed
+
+
+feed.main()
```

Comment:
I'm not sure I understand in which case this is needed

Reply:
Having a `__main__.py` file in the module allows to call the module like a command: `python -m dakara_feeder --help`. See this [blog post](https://shaneoneill.io/2019/06/12/use-__main__-py/) for more details. I use `__main__.py` as a complement to the command defined in `setup.cfg`, as it is more recommended (see this [blog post](http://sametmax.com/lancer-correctement-python-et-ses-commandes-cousines/)).

I have a doubt, though: if there were several commands, we should indicate which one is requested (it would be like `python -m dakara_feeder feed` and `python -m dakara_feeder do-something-else`). For now there is only one command (and I doubt there could be more), so I did not feel the urge to add such a mechanism. Do you think it is fine?

Reply:
This file is currently not testable (as it just execute one command), so I will ignore absence of coverage and Codecov warning.